pom.xml

```
@@ -88,6 +88,18 @@
           <version>4.12</version>
           <scope>test</scope>
         </dependency>
+       <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.10.19</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

A slightly newer version is available (if suitable).
```suggestion
            <version>4.13</version>
```

---

Indeed.